grape-entity.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = 'grape-entity'
 
-  s.add_runtime_dependency 'activesupport'
+  s.add_runtime_dependency 'activesupport', '<= 4.2.6'
```

Comment:
@dblock  We have to make a choice what is more important: supporting old rubies or supporting last AS/rack/json.